packages/angular/projects/ui-angular/src/lib/services/__tests__/mockAuthService.ts

```diff
@@ -0,0 +1,22 @@
+// mock state machine service
```

Comment:
Unlike vue/angular, Angular was using `xstate` directly without any util library (`@xstate/react`), so had to do mocks at xstate level.